src/Exception/AccessDeniedHttpException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException as BaseAccessDeniedHttpException;
+
+/**
+ * This exception is thrown when the access to a resource has been denied.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class AccessDeniedHttpException extends BaseAccessDeniedHttpException
```

Comment:
It's somewhat strange we do handle 404 in our code to render the Contao page, but we don't do the same for a 403. What does a developer do if he wants to disallow access to a page? Throwing an `AccessDeniedHttpException` would be wrong I guess?